### PR TITLE
feat: Add initial version of data-prepper helm chart

### PR DESCRIPTION
**Pull request by SergK**
### Description
Let's start with the initial version of the data-prepper.
We can extend it later with more examples of configuration
 
### Issues Resolved
Related issue: #45
 
### Check List
- [x] Commits are signed per the DCO using --signoff

For any changes to files within Helm chart directories:
- [x] Helm chart version bumped
- [ ] Helm chart `CHANGELOG.md` updated to reflect change

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/helm-charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by SergK** [comment]
Hello @TheAlgo, any chance to push this forward?

**Comment by ruanyl** [comment]
@SergK I'm getting error log `No configuration found for '2cdf8d8a' at 'null' in 'null'`, this seems relate to log4j configuration, have you seen the error?

**Comment by SergK** [comment]
@ruanyl yep, fix is on the way in a 20 minutes

**Comment by SergK** [comment]
@ruanyl I've fixed the issue

**Comment by SergK** [comment]
fixed ci issue

**Comment by ruanyl** [comment]
> @ruanyl I've fixed the issue

Tried another time and it works fine for me 👍 

@TheAlgo Could we get more reviewers on this PR?

**Comment by peterzhuamazon** [comment]
Thanks @SergK for contribution.
Adding @TheAlgo @DandyDeveloper @prudhvigodithi to take a look.


Thanks.

**Comment by prudhvigodithi** [comment]
Thanks @SergK I will take the opportunity to review this and we can ship this chart.
Thank you

**Comment by prudhvigodithi** [comment]
Thanks for your contribution @SergK the initial version of the chart LGTM, please add the `CHANGELOG.md ` and `README.md` files similar to what we have for other charts in this repo.
Thanks 
@getsaurabh02

**Comment by prudhvigodithi** [comment]
We can move forward with this PR once we have the `CHANGELOG.md`  and `README.md` files added and also I see no blocker for releasing this helm chart (using https://github.com/opensearch-project/helm-charts/blob/main/.github/workflows/release.yaml) and publishing to https://artifacthub.io/packages/search?org=opensearch-project&sort=relevance&page=1.

**Comment by SergK** [comment]
@prudhvigodithi done

File: charts/data-prepper/README.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+This command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Contributing
+
+We welcome contributions! Please read our [CONTRIBUTING.md](../../CONTRIBUTING.md) for details on how to submit contributions to this project.
+
+## Maintainers
+
+| Name | Email | Url |
```
Comment (prudhvigodithi) [inline charts/data-prepper/README.md:61]: @SergK can you please fill the other details in this table ?
Please refer https://github.com/opensearch-project/helm-charts/blob/main/MAINTAINERS.md

File: charts/data-prepper/README.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
+# Data Prepper Helm Chart
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.8.0](https://img.shields.io/badge/AppVersion-2.8.0-informational?style=flat-square)
+
+A Helm chart for Data Prepper
+
+**Homepage:** <https://opensearch.org>
```
Comment (prudhvigodithi) [inline charts/data-prepper/README.md:7]: How about we have https://opensearch.org/docs/latest/data-prepper/ as homepage?

**Comment by TheAlgo** [comment]
Thanks @SergK for the contribution and apologies for late review. I will review it sometime today.

File: charts/data-prepper/README.md (hunk 1 above)
Comment (SergK) [reply charts/data-prepper/README.md:61]: done

File: charts/data-prepper/README.md (hunk 2 above)
Comment (SergK) [reply charts/data-prepper/README.md]: done

**Review by TheAlgo — APPROVED** [review]
Should we add a CI similar to opensearch and opensearch-dashboards here?

**Comment by prudhvigodithi** [comment]
> Should we add a CI similar to opensearch and opensearch-dashboards here?

Thanks @TheAlgo yes eventually we need them @SergK.

**Review by prudhvigodithi — APPROVED** [review]

**Comment by prudhvigodithi** [comment]
The release is successful https://github.com/opensearch-project/helm-charts/releases/tag/data-prepper-0.1.0 
```
helm search repo opensearch/data-prepper
NAME                   	CHART VERSION	APP VERSION	DESCRIPTION                  
opensearch/data-prepper	0.1.0        	2.8.0      	A Helm chart for Data Prepper
```
Will wait to get the artifacthub updated https://artifacthub.io/packages/search?repo=opensearch-project-helm-charts&sort=relevance&page=1
Thanks @SergK

**Pull request merged**